AmplifyPlugins/Core/AWSPluginsCore/Model/GraphQLRequest/GraphQLRequest+Model.swift

```diff
@@ -101,17 +186,8 @@ extension GraphQLRequest {
                                    decodePath: document.name + ".items")
     }
 
-    /// Creates a `GraphQLRequest` that represents a subscription of a given `type` for a `model` type.
-    /// The request will be created with the correct document based on the `ModelSchema`.
-    ///
-    /// - Parameters:
-    ///   - modelType: the metatype of the model
-    ///   - type: the subscription type, either `.onCreate`, `.onUpdate` or `.onDelete`
-    /// - Returns: the `GraphQLRequest` ready to be used
-    ///
-    /// - seealso: `GraphQLSubscription`, `GraphQLSubscriptionType`
     public static func subscription<M: Model>(of modelType: M.Type,
-                                              type: GraphQLSubscriptionType) -> GraphQLRequest<M> {
+                                              _ type: GraphQLSubscriptionType) -> GraphQLRequest<M> {
```

Comment:
It's a bit unusual to see an API method with a named first parameter and unnamed second one (that's not a trailing closure). That means the call site would look something like `.subscription(of: Post.self, .create)`? I appreciate the brevity, but it feels a bit wonky.

Reply:
fair point, I can definitely change it back.

The call looks like this:

```swift
.subscription(of: Post.self, .onCreate)
```

The `on` prefix in the enum motivated me to make the change, but I think I prefer to have it named as well.

Reply:
changed back to the named `type`